src/QUMainWindow.cpp

```diff
@@ -31,6 +31,12 @@
 #include "compact_lang_det.h"
 #include "srtparser.h"
 
+std::vector<std::pair<std::string, const char *>> FILENAME_REPLACEMENTS{
+	std::make_pair(std::string("?:\""), ""),
+	std::make_pair(std::string("<"), "("),
+	std::make_pair(std::string(">"), ")"),
+	std::make_pair(std::string("/\\|*"), "-")};
+

```

Comment:
Wasn't sure where to put a constant like this one @bohning , would you prefer this in a different place?

Reply:
I guess it's fine where it is. The whole Creator code could benefit from some refactoring and splitting into multiple files, but for the status now, I think it's okay to put it right there.